src/matrix_graph.rs

```diff
@@ -128,10 +128,10 @@ impl<T: Zero> Nullable for NotZero<T> {
     }
 }
 
-impl<T: Zero> Into<Option<T>> for NotZero<T> {
```

Comment:
The CI is breaking here because this is only allowed since 1.41 [1].
We are going to raise the MSRV (#428), so it shouldn't be a problem.

[1]: https://blog.rust-lang.org/2020/01/30/Rust-1.41.0.html#relaxed-restrictions-when-implementing-traits).